src/plugins/discover/public/application/components/sidebar/lib/visualize_trigger_utils.ts

```diff
@@ -95,8 +95,14 @@ export async function isFieldVisualizable(
   indexPatternId: string | undefined,
   contextualFields: string[]
 ) {
-  if (field.name === '_id' || !indexPatternId) {
+  const notVisualizableFieldTypes: string[] = [KBN_FIELD_TYPES.UNKNOWN, KBN_FIELD_TYPES.CONFLICT];
+  if (
+    field.name === '_id' ||
+    notVisualizableFieldTypes.includes(field.spec.type) ||
+    !indexPatternId
+  ) {
```

Comment:
I think we should think about migrating parts of it to the `index_pattern_fields.ts` later on (think it's not only valid in Discover?), to have it more centralized, currently `visualizable()` returns true, if aggregatable
https://github.com/elastic/kibana/blob/9450248ebede86b0cb515bbd1a96046268f4a332/src/plugins/data/common/index_patterns/fields/index_pattern_field.ts#L131-L133